templates/microsoft_cloud_for_industry/financial_services_landing_zone/variables.tf

```diff
@@ -0,0 +1,327 @@
+    enableBgpRouteTranslationForNat = optional(bool)
+    enableDnsForwarding             = optional(bool)
+    asn                             = optional(number)
+    bgpPeeringAddress               = optional(string)
+    peerWeight                      = optional(number)
+    vpnClientConfiguration = optional(object({
+      vpnAddressSpace = optional(list(string))
+    }), {})
+  })
+  default     = { "name" : "noconfigVpn" }
+  description = "(Optional) configuration options for the VPN Gateway."
+}
+
+variable "deploy_bastion" {
+  type        = bool
+  default     = true
+  description = "Toggles deployment of Azure Bastion. True to deploy, otherwise false. (e.g true)"
+}
+
+variable "landing_zone_management_group_children" {
+  type = map(object({
+    id          = string
+    displayName = string
+  }))
+  default     = {}
+  description = "(Optional) array of child management groups to deploy under the FSI Landing Zone's management group."
+}
+
+# tflint-ignore: terraform_unused_declarations
+variable "architecture_definition_template_path" {
+  type        = string
+  default     = ""
+  description = "The path to the architecture definition template file to use."
+}
+
+# tflint-ignore: terraform_unused_declarations
+variable "architecture_definition_override_path" {
+  type        = string
+  default     = ""
+  description = "The path to the architecture definition file to use instead of the default."
+}
+
+# tflint-ignore: terraform_unused_declarations
+variable "apply_alz_archetypes_via_architecture_definition_template" {
+  type        = bool
+  default     = true
+  description = "Toggles assignment of ALZ policies. True to deploy, otherwise false. (e.g true)"
+}
+
+variable "ms_defender_for_cloud_email_security_contact" {
+  type        = string
+  default     = "security_contact@replaceme.com"
+  description = "An e-mail address that you want Microsoft Defender for Cloud alerts to be sent to."
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$", var.ms_defender_for_cloud_email_security_contact))
+    error_message = "The email address must be a valid format."
+  }
+}
+
+variable "bastion_outbound_ssh_rdp_ports" {
+  type        = set(string)
+  default     = ["22", "3389"]
+  description = "Array of outbound destination ports and ranges for Azure Bastion."
+}
+
+variable "policy_effect" {
+  type        = string
+  default     = "Deny"
+  description = "The policy effect used in all assignments for the Sovereignty Baseline policy initiatives."
+  validation {
+    condition     = contains(["Audit", "Deny", "Disabled"], var.policy_effect)
+    error_message = "Allowed values are 'Audit', 'Deny', and 'Disabled'."
+  }
+}
+
+variable "policy_assignment_enforcement_mode" {
+  type        = string
+  default     = "Default"
+  description = "The enforcement mode used in all policy and initiative assignments."
+}
+
+variable "deploy_log_analytics_workspace" {
```

Comment:
We discussed removing this option since the Landing Zone would not meet standard without it.